app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -72,6 +76,10 @@ class RecentPatientsViewController @Inject constructor(
     return Observable.merge(updateRecentPatients, toggleEmptyState)
   }
 
+  private fun maybeAddSeeAllButton(recentPatients: List<RecentPatientItem>, config: PatientConfig) =
+      if (recentPatients.size > config.recentPatientLimit) recentPatients + SeeAllItem
+      else recentPatients
```

Comment:
We should add brackets to the `if-else` statements even if they are one lines only.

Reply:
If there are multiple lines in a method then we should avoid using `=` in a method and enclose the statements in `{ }`

Reply:
Inlined